demisto_sdk/commands/find_dependencies/tests/find_dependencies_test.py

```diff
@@ -178,6 +178,41 @@ def test_collect_scripts_depends_on_two_scripts(self, id_set):
 
         assert IsEqualFunctions.is_sets_equal(found_result, expected_result)
 
+    def test_collect_scripts__filter_toversion(self, id_set):
+        """
+        Given
+            - A script entry in the id_set depending on QRadar command.
+
+        When
+            - Building dependency graph for pack.
+
+        Then
+            - Extracting the packs that the script depends on.
+            - Should ignore the Deprecated pack due to toversion settings of old QRadar integration.
```

Comment:
Should ignore Deprecated packs or `NonSupported` pack?

Reply:
deprecated, old integration is there